phpBB/phpbb/notification/manager.php

```diff
@@ -834,7 +846,9 @@ public function get_default_methods()
 	/**
 	 * Helper to get the notifications item type class and set it up
 	 *
-	 * @return type\type_interface
+	 * @param string $notification_type_name
+	 * @param array  $data
+	 * @return \phpbb\notification\type\type_interface
```

Comment:
`method\method_interface`

You need to be consistent actually, pick one form and use it everywhere